server/index.js

```diff
@@ -5,7 +5,7 @@ require('dotenv/config');
 
 const app = express();
 const server = http.Server(app);
-const io = require('socket.io')(server);
+global.io = require('socket.io')(server);
```

Comment:
Why global?

Reply:
I am using it in socketService file so made it gobal.